site/en/blog/colrv1-fonts/index.md

```diff
@@ -0,0 +1,301 @@
+takes about 1.2 MB inflated, but is about 0.6MB in woff2 form. A build of the
+full Noto Emoji Glyph set is reduced from 9MB for the bitmap version to 1.85MB
+in COLRv1+woff2 form.
+
+<figure>
+{% Img src="image/roKb5a4ddjOSoIbxRFI9kDeuVvE2/OULzoVLf1nYWZCaZWhBa.png",
+alt="Bar chart comparing Noto Emoji as Bitmap font and COLRv1 font, ~9MB
+vs. 1.85MB", width="800", height="495" %}
+<figcaption>Noto Emoji font size CBDT/CBLC vs. COLRv1 after WOFF2 compression.
+</figcaption>
+</figure>
+
+## Color Font Use Cases
+
+### Catchy Headlines
+
+A fresh color font makes visual highlights, headlines, banners really pop out.
+
+_TODO: Replace image with Plakato Glitch_
+
+<figure> {% Img
+src="image/roKb5a4ddjOSoIbxRFI9kDeuVvE2/y9DYZuiXxdGshTyTJnSI.png", alt="Example
+text styled with the Plakato font, which uses basic geometric shapes to build
+letters, here featuring pink to red and purple sweep gradient.", width="800",
+height="559" %} <figcaption><a
+href="https://www.underware.nl/blog/2022/01/plakato-color/">Plakato COLRv1</a>
+featuring energetic sweep gradients, made by the innovative type foundry <a
+href="https://underware.nl">Underware</a>, (<a
+href="https://twitter.com/underware">@underware</a> on Twitter). Read more about Underware’s
+first COLRv1 release in their <a
+href="https://www.underware.nl/blog/2022/01/plakato-color/">blog post</a>.
+</figcaption> </figure>
+
+
+### No more image replacement: Emoji Fonts
+
+If you support user generated content, your users probably use emoji. Today it's
+very common to scan text and replace any emoji encountered with images to ensure
+consistent cross-platform rendering and the ability to support newer emojis than
+the OS supports. Those images then have to be switched back to text during
+clipboard operations. Here's a real example:
+
+<figure>
+{% Img src="image/roKb5a4ddjOSoIbxRFI9kDeuVvE2/dgpLYdWxn0G94lH0rVWI.png", alt="A
+code snippet showing inline images as img tags and metadata as part of a chat history", width="800", height="88" %}
+<figcaption>Image replacement in Google Chat</figcaption>
+</figure>
+
+If you have an emoji font you just render the text in the font, like this:
+
+```html
+<style>
+@import url(https://fonts.googleapis.com/css2?family=Noto+Color+Emoji);
+
+.emoji {
+  font-family: "Noto Color Emoji", sans-serif;
+  font-size: 6em;
+}
+</style>
+<span class="emoji">🙂</span>
+```
+
+Similarly, in an emoji reaction component, COLRv1 offers the opportunity to use
+a compact font file instead of a catalog of image assets.
+
+<figure> {% Img
+src="image/roKb5a4ddjOSoIbxRFI9kDeuVvE2/dgPBLfMKknICwHOyNgrI.png", alt="Emoji
+picker UI as used on GitHub", width="408", height="390" %} <figcaption>Emoji
+Reaction Picker on GitHub</figcaption> </figure>
+
+Imagine how many images you'd have to fetch for a complete emoji picker!
+
+### Color in Icon Fonts
+
+Using color in icon fonts adds clarity through layers and makes glyphs easier to understand.
```

Comment:
What does 'clarity through layers' me4an?

Reply:
I meant something like: "colors create visual layers and give more depth to the glyph" and thus make it easier to understand, but maybe that's a bit esoteric. I opted to remove "through layers".